### PR TITLE
Create bounty.lua

### PR DESCRIPTION
**Pull request by ARGO-Games**
Introduce a Bounty System.

### CODE REVIEW

**Review by Dutch-VanDerLinde — APPROVED** [review]

File: Lua/bounty.lua [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    killers.RName[killerName] = killerName
+    killers.CName[killerName] = Util.FindClientCharacter(killerName)
+    killers.Count[killerName] += 1
+end)
+
+Traitormod.AddCommand("!claim", function (client, args) 
+    local found = nil
+    local bounty = nil
+    for i=1, killers.CName do
+        if killers.CName[i] == client.Name then
```
Comment (Dr-javier) [inline Lua/bounty.lua]: if killers.CName[i] == client then

Comment (Dr-javier) [reply Lua/bounty.lua]: this is because Util.FindClientCharacter(killerName) returns client and not client.Name

**Review by Dr-javier — CHANGES_REQUESTED** [review]
need to add dofile(Traitormod.Path .. "/Lua/bounty.lua") in traitormod.lua

File: Lua/bounty.lua (hunk 1 above)
Comment (Dutch-VanDerLinde) [inline Lua/bounty.lua]:
```suggestion
        if killers.CName[i] == client then
```

**Review by Dutch-VanDerLinde — CHANGES_REQUESTED** [review]

**Review by Dutch-VanDerLinde — APPROVED** [review]

**Comment by Dutch-VanDerLinde** [comment]
code is good, javier is testing and then we'll merge

**Review by Dr-javier — APPROVED** [review]

**Pull request merged**